examples/migrate_data_volume_from_bay1_to_bay12.yml

```diff
@@ -0,0 +1,139 @@
+    profile_name1: "profile_san"
+    server_hardware_name2: "2-USE722C99W-M, bay 12"
+    server_profile_template: "Windows2016_i3s"
+    profile_name2: "profile_i3s"
+    management_network: "Management"
+    SAN_A_network: "SAN-A"
+    SAN_B_network: "SAN-B"
+    storage_pool_name_A: "Pool name A"
+    storage_pool_name_B: "Pool name B"
+
+  tasks:
+
+    - name : "Create/Update Server Profile - {{ profile_name1 }}"
+      oneview_server_profile:
+        config: "{{ config }}"
+        data:
+          name: "{{ profile_name1 }}"
+          serverHardwareTypeName: "{{ server_hardware_type_name }}"
+          enclosureGroupName: "{{ enclosure_group_name }}"
+          server_hardware: "{{ server_hardware_name1 }}"
+          serialNumberType: Virtual
+          iscsiInitiatorNameType: AutoGenerated
+          macType: Virtual
+          wwnType: Virtual
+          description: 'use case 1: boot from SAN'
+          affinity: Bay
+          connections:
+          - id: 1
+            name: LAB
+            functionType: Ethernet
+            portId: Mezz 3:2-a
+            requestedMbps: '2500'
+            networkName: "{{ management_network }}"
+            requestedVFs: '0'
+            boot:
+              priority: NotBootable
+          - id: 2
+            name: SAN-A
+            functionType: FibreChannel
+            portId: Mezz 2:1
+            requestedMbps: Auto
+            networkName: "{{ SAN_A_network }}"
+            boot:
+              priority: Primary
+              bootVolumeSource: ManagedVolume
+          - id: 3
+            name: SAN-B
+            functionType: FibreChannel
+            portId: Mezz 2:2
+            requestedMbps: Auto
+            networkName: "{{ SAN_B_network }}"
+            boot:
+              priority: NotBootable
+          boot:
+            manageBoot: true
+            order:
+            - CD
+            - USB
+            - HardDisk
+            - PXE
+          bootMode:
+            manageMode: true
+            mode: BIOS
+          bios:
+            manageBios: false
+            overriddenSettings: []
+          hideUnusedFlexNics: true
+          iscsiInitiatorName: ''
+          localStorage:
+            sasLogicalJBODs: []
+            controllers: []
+          sanStorage:
+            hostOSType: Windows Server 2016
+            manageSanStorage: true
+            volumeAttachments:
+            - id: 1
+              isBootVolume: true
+              lunType: Auto
+              storagePaths:
+              - isEnabled: true
+                connectionId: 2
+              - isEnabled: true
+                connectionId: 3
+              volumeName: win_boot
+              volumeStoragePoolName: "{{ storage_pool_name_A }}"
+      delegate_to: localhost   
+
+    - name : "Create Server Profile - {{ profile_name2 }}"
+      oneview_server_profile:
+        config: "{{ config }}"
+        data:
+          server_hardware: "{{ server_hardware_name2 }}"
```

Comment:
This will not work anymore as server_hardware was not following the style guide from other fields. It should be 'serverHardwareName' now.

Reply:
Fixed.